framework/scene_graph/components/image.cpp

```diff
@@ -312,10 +319,25 @@ std::vector<uint8_t> &Image::get_mut_data()
 	return data;
 }
 
+void Image::update_hash()
+{
+	data_hash = 0;
+	for (size_t i = 0; i < data.size(); ++i)
+	{
+		glm::detail::hash_combine(data_hash, data[i]);
```

Comment:
`glm::detail::hash_combine` works on hashes of type `size_t`. Maybe it's worth to loop by 4 bytes, instead of 1?

Reply:
Done